text/rocq-roadmap.md

```diff
@@ -5,7 +5,7 @@ Rocq Long Term Roadmap
 
 The Rocq Development Team, building on 40 years of experience with Coq, is pleased to announce the Rocq Prover, the next-generation proof assistant. The Rocq Prover is designed to empower a diverse range of users, from mathematicians seeking formal rigor to software engineers building high-assurance systems.
 
-While Rocq leverages the rich heritage of Coq, it embraces a more agile development philosophy. This allows us to deliver impactful changes more frequently, accelerating innovation and user experience improvements.
+Rocq builds upon the rich heritage and diverse ecosystem of Coq. We promote a development methodology which achieves the delicate balance of being mindful of compatibility and ecosystem maintainability, while sufficiently agile to deliver impactful changes frequently, boosting research and innovation, and continuously improving user experience.
```

Comment:
I would split this in shorter sentences.

> Rocq {builds upon, leverages} the rich heritage and diverse ecosystem of Coq; it is mindful of compatibility and ecosystem maintainability. It also keeps evolving; we deliver impactful changes frequently, practice active research, and continuously improve the user experience.

Reply:
So you still want to remove the word "agile" in this context?

Reply:
"agile" and rings bullshit-bells to me; I think that you are less sensitive about it because you talk to software-engineering people, but 99% of the times I hear this word it is in the context of someone spewing nonsense. Same with "innovation". Personally I would avoid those words because of their largely negative connotations among researchers (which remain the main audience of Coq).

Reply:
Oh right, I had not noticed that you were also removing the word "innovation".

I think we are not going to converge on something that satisfies us both here, so maybe we should bring this topic back to the next Coq Call?

Reply:
Regarding the use of "agile" by @erikmd, it was to mention the advanced use of tooling and the regular developer meeting. This can probably be said explicitly some way or another without using the word "agile".

As for "innovation", isn't it already morally covered by the words "research", "impactful", "improve", "evolving"?

Of course, it depends also on the audiences we want to target.

Reply:
```suggestion
Rocq builds upon the rich heritage and diverse ecosystem of Coq. We promote a development methodology which achieves the delicate balance of being mindful of compatibility and ecosystem maintainability, while sufficiently fast-paced to deliver impactful changes frequently, boosting academic and industrial research, and continuously improving user experience.
```

Since so many people dislike the words "agile" and "innovation", this is my proposal to remove them without changing the meaning.

There was also an issue with a too-long sentence, so this could become:

```suggestion
Rocq builds upon the rich heritage and diverse ecosystem of Coq. We promote a fast-paced development methodology to deliver impactful changes frequently, supporting academic and industrial research, and continuously improving user experience. At the same time, we are mindful of compatibility and ecosystem maintainability.
```

I personally liked my first proposal better than the second one, and the second one better than the third one, but if there is a clear preference for any of these, I'm fine with them. If this is still not enough and people do not think this is a sufficient improvement on the current status, then I will not bother further and close this PR, to let others do their own attempts.

Reply:
Yes, I do think there is a significant benefit in removing those loaded words to sound "less bullshit". Either proposals are fine, I think that the last one reads better due to having shorter sentences. Thanks!